templates/options.php

```diff
@@ -4,6 +4,7 @@
  */
 ?>
 <div class="wrap">
+	<?php screen_icon(); ?>
```

Comment:
Believe this function [is deprecated](http://codex.wordpress.org/Function_Reference/screen_icon).